index.html

```diff
@@ -7,10 +7,81 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Texturina:wght@100&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Bungee+Outline&display=swap" rel="stylesheet">
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <header>
+        <div class="header-left">
+            <img src="design/cake-logo-transparent.png" alt="We teach cakes!" class="logoImage"/>
+        </div>
+        <div class="header-right"> 
+            <div class="mobile-menu">
+                <div class="menu"></div>
```

Comment:
Nice touch using css to style your burger icon 👏 Considering this is just a decorative element I would choose to use a `<span>` tag.